### PR TITLE
Feat: Added Avatar Components

### PR DESCRIPTION
**Pull request by shuntzhere**
### Added multiple avatars including :
- Standard Avatars
- Hoverable Avatars
- Multiple sized avatars that include
xs, sm, md, lg, xl and xxl

https://user-images.githubusercontent.com/47948956/153047314-cb51a190-5ac8-4f23-9247-e3412639a2c2.mp4

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  height: 10px;
+}
+
+.sm {
+  width: 20px;
+  height: 20px;
+}
+
+.md {
+  width: 30px;
```
Comment (SimranjitKaurMaan) [inline components/avatar/avatar.css:32]: You could use rem instead.

**Review by SimranjitKaurMaan — COMMENTED** [review]
Overall lgtm. Just a small suggestion.

File: components/avatar/avatar.css (hunk 1 above)
Comment (shuntzhere) [reply components/avatar/avatar.css:32]: Hey, thanks for reviewing, it's just a choice a made to fit my design better.

Comment (SimranjitKaurMaan) [reply components/avatar/avatar.css:32]: Sure.

**Review by SimranjitKaurMaan — APPROVED** [review]

**Review by coldpigli — APPROVED** [review]
Everything looks fine to me. You can go ahead and merge.

File: sidebar.css [hunk 2]
```diff
@@ -5,7 +5,7 @@
 }
 
 .docs-content {
-  margin-top: 2rem;
+  margin: 0 0 2rem 0;
```
Comment (ttahm3d) [inline sidebar.css:8]: Use the margin-bottom directly.

**Review by ttahm3d — APPROVED** [review]
Looks all good.

File: sidebar.css (hunk 2 above)
Comment (shuntzhere) [reply sidebar.css:8]: Update these values, Thank You.

**Pull request merged**